src/tests/DateTimeTests.fs

```diff
@@ -373,12 +373,10 @@ let ``DateTime Inequality works``() =
 
 [<Test>]
 let ``TimeSpan constructors work``() =
-    let t1 = TimeSpan(20000L)
-    let t2 = TimeSpan(3, 3, 3)
-    let t3 = TimeSpan(5, 5, 5, 5)
-    let t4 = TimeSpan(7, 7, 7, 7, 7)
-    t1.TotalMilliseconds + t2.TotalMilliseconds + t3.TotalMilliseconds + t4.TotalMilliseconds
-    |> equal 1091715009.0
+    TimeSpan(20000L).TotalMilliseconds |> equal 2.0
+    TimeSpan(3, 3, 3).TotalMilliseconds |> equal 10983000.0
+    TimeSpan(5, 5, 5, 5).TotalMilliseconds |> equal 450305000.0
+    TimeSpan(7, 7, 7, 7, 7).TotalMilliseconds |> equal 630427007.0
```

Comment:
Again, I'd prefer if you add extra assertions instead of replacing the old one :+1: